tests/unit/components/setup.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * The tests for our components need React, because we are using the new
+ * technology of shallow rendering, so we are passing to the renderer
```

Comment:
Yes, but this is a consequence of using the `Shallow Renderer`, that we are using JSX. But ok, I will update it.